app/models/UserAddress.scala

```diff
@@ -21,7 +21,15 @@ import play.api.libs.json.{Json, OFormat}
 case class UserAddress(property: Option[String],
                        street: String,
                        town: String,
-                       postcode: String)
+                       postcode: String) {
+  def toCheckYourAnswerFormat: String =
+    if (property.isDefined) {
+      s"${property.getOrElse("")} $street\n$town\n$postcode"
```

Comment:
I don't think this actually separates the lines once it gets to the page

![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/31283663/c5ebcbc3-012e-48a3-8aa4-960426da3cd4)